tests/node_eviction_test.go

```diff
@@ -0,0 +1,262 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+const (
+	cirrosImageDev  string = "registry:5000/kubevirt/cirros-registry-disk-demo:devel"
+	cirrosImageTest string = "kubevirt/cirros-registry-disk-demo:latest"
+)
+
+type vmi struct {
+	name     string
+	filename string
+	url      string
+}
+
+var _ = Describe("Node Eviction", func() {
+
+	flag.Parse()
+
+	var (
+		simpleVmi   vmi
+		vmBackedVmi vmi
+		rsBackedVmi vmi
+	)
+
+	BeforeEach(func() {
+		ktests.BeforeTestCleanup()
+
+		args := []string{"get", "node", "--selector=node-role.kubernetes.io/compute=true", "--output=jsonpath={.items..metadata.name}"}
+		output, err := ktests.RunOcCommand(args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		var count int
+		nodes := strings.Fields(output)
+		for _, node := range nodes {
+			args := []string{"get", "node", node}
+			output, err := ktests.RunOcCommand(args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			if !strings.Contains(output, "SchedulingDisabled") {
+				count += 1
+			}
+		}
+		if count < 2 {
+			panic(fmt.Errorf("not enough schedulable nodes for this testing"))
+		}
+
+		simpleVmi = vmi{
+			name:     "vmi-ephemeral",
+			filename: "/tmp/vmi-ephemeral.yaml",
+			url:      "https://raw.githubusercontent.com/kubevirt/kubevirt/master/cluster/examples/vmi-ephemeral.yaml",
+		}
+
+		vmBackedVmi = vmi{
+			name:     "vm-cirros",
```

Comment:
same. we need to be able to run on different OSs.
id prefer that we will have a few yamls in this repo (tests/manifests), with everything we need,
and to reuse these in multiple tests

Reply:
The problem to have the yaml in the repo is that, it need to maintain the yamls from time to time, to pull it from kubevirt/kubevirt can ensure it's latest, and it's a test to ensure latest manifests works.